gemini/sample-apps/gemini-mesop-cloudrun/README.md

```diff
@@ -0,0 +1,100 @@
+# Cloud Run application utilizing Mesop that demonstrates working with Vertex AI Gemini API
+
+|           |                                                |
+| --------- | ---------------------------------------------- |
+| Author(s) | [Hussain Chinoy](https://github.com/ghchinoy) |
+
+This application demonstrates a Cloud Run application that uses the [Mesop](https://github.com/google/mesop) UI framework.
+
+Sample screenshots and video demos of the application are shown below:
+
+## Application screenshots
+
+![image playground](images/imageplayground.png)
```

Comment:
```suggestion
![image playground](https://storage.googleapis.com/github-repo/generative-ai/sample-apps/mesop-cloudrun/imageplayground.png)
```

Recommended to put images in GCS instead of the repo